libs/ui-watt/src/lib/components/modal/watt-modal.component.html

```diff
@@ -0,0 +1,21 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-template #modal>
+  <h2 class="watt-modal-title">{{ title }}</h2>
```

Comment:
question: Watt applies default styling on the `h2` by tag and class (see `ui-watt\src\lib\foundations\typography\index.scss`). As part of that styling things like `color` and `font-size`/`font-weight` are also set. We're also adjusting these in the `watt-modal.component.scss` file. Is it a requirement from UX to change these things or can we leave the "default" styling?

Reply:
Fixed by adding a new large h3 variant and using that in the modal.